usage.md

```diff
@@ -266,4 +266,22 @@ interface SwitchEthereumChainParameter {
 
 If the error code (error.code) is 4902, then the requested chain has not been added, and you have to request to add it via `wallet_addEthereumChain`.
 
+### Track New Assets
+
+#### `wallet_watchAsset`
+
+This method is specified by [EIP-747](https://eips.ethereum.org/EIPS/eip-747).
+
+```ts
+params: {
+        type: 'ERC20',
+        options: {
+          address: '0xB983E01458529665007fF7E0CDdeCDB74B967Eb6',
+          symbol: 'FOO',
+          decimals: 18,
+          image: 'https://foo.io/token-image.svg',
+        },
+      },
```

Comment:
Please use
```js
await provider.request({
  method: 'wallet_watchAsset',
  params: {
    type: 'ERC20',
    options: {
      address: '0xB983E01458529665007fF7E0CDdeCDB74B967Eb6',
      symbol: 'FOO',
      decimals: 18,
      image: 'https://foo.io/token-image.svg',
    },
  },
});
```

Reply:
Not part of Beta 1.2 release Auth SDK 0.2.2 so closing this hotfix PR.